types/mydecimal.go

```diff
@@ -2291,15 +2293,19 @@ func NewDecFromUint(i uint64) *MyDecimal {
 func NewDecFromFloatForTest(f float64) *MyDecimal {
 	dec := new(MyDecimal)
 	err := dec.FromFloat64(f)
-	terror.Log(errors.Trace(err))
+	if err != nil {
+		log.Panic("encountered error", zap.Error(err), zap.String("DecimalStr", strconv.FormatFloat(f, 'g', -1, 64)))
```

Comment:
how about `log.Errorf()`?

Reply:
We should panic here when error occurs, so that the unit test can fail fast instead of getting a wrong value and throw a misunderstanding error.

Reply:
This function is used only for unit test. If some error happens, it should panic to notify the unit test to fail.

Reply:
OK